tests/__init__.py

```diff
@@ -301,6 +301,37 @@ def test_dependencies(self):
                 ("aloha_world",),
             )
 
+    def test_uids(self):
```

Comment:
Just to make clear the purpose of the test:

```suggestion
    def test_uids_modified(self):
```

Reply:
i renamed it to test_modified_uids (to be consistent with test_default_uids)